Assets/Script/Augment/AugmentImg.cs

```diff
@@ -0,0 +1,189 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Reflection;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class AugmentImg : MonoBehaviour
+{
+    public Sprite augImg1;
+    public Sprite augImg2;
+    public Sprite augImg3;
+    public Sprite augImg4;
+    public Sprite augImg5;
+    public Sprite augImg6;
+    public Sprite augImg7;
+    public Sprite augImg8;
```

Comment:
```suggestion
    public Sprite[] augImages;
```
ถ้าจำไม่ผิด เราสามารถสร้างเป็น Array แล้วสามารถใส่ Sprite GameObject เป็น Dropdown ได้เลย แต่ถ้าไม่ได้ก็เอาแบบนี้ครับ